rs-commerce/coverage/App.tsx.html

```diff
@@ -0,0 +1,184 @@
+
+<!doctype html>
+<html lang="en">
+
+<head>
+    <title>Code coverage report for App.tsx</title>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="prettify.css" />
+    <link rel="stylesheet" href="base.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.png" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <style type='text/css'>
```

Comment:
Comment fot this file and all files from folder `coverage`. Coverage folder was added in `gitignor` file after pushing it in git. You need to delete all this files from git in github manually to avoid existing this files in develop branch